### PR TITLE
update compress support compressindex

### PR DESCRIPTION
**Pull request by owen200008**
update compress support compressindex

### CODE REVIEW

File: extensions/assets-manager/AssetsManagerEx.cpp [hunk 1]
```diff
@@ -49,6 +49,10 @@ NS_CC_EXT_BEGIN
 const std::string AssetsManagerEx::VERSION_ID = "@version";
 const std::string AssetsManagerEx::MANIFEST_ID = "@manifest";
 
+bool SortCompressFileIndexAssert(const CompressedFilesInfoAsserts& a, const CompressedFilesInfoAsserts& b)
```
Comment (minggo) [inline extensions/assets-manager/AssetsManagerEx.cpp:52]: please put it into a namespace like
```c++
namespace {
 bool SortCompressFileIndexAssert(const CompressedFilesInfoAsserts& a, const CompressedFilesInfoAsserts& b) {
    ...
}
}
```

**Comment by minggo** [comment]
will it break compatibility?

File: tools/tolua/cocos2dx_studio.ini [hunk 2]
```diff
@@ -53,7 +53,7 @@ skip =  *::[^visit$ copyWith.* onEnter.* onExit.* ^description$ getObjectType .*
         ActionObject::[initWithDictionary initWithBinary],
         BaseData::[copy subtract],
         ActionTimelineCache::[getInstance loadActionTimelineFromXML loadAnimationWithDataBuffer createActionWithDataBuffer],
-        ActionTimeline::[setFrameEventCallFunc]
```
Comment (minggo) [inline tools/tolua/cocos2dx_studio.ini:56]: is it something related with this PR? If not, i think you should create a another branch for it.

File: extensions/assets-manager/AssetsManagerEx.cpp (hunk 1 above)
Comment (owen200008) [reply extensions/assets-manager/AssetsManagerEx.cpp:52]: it is under  namespace cocos2d { namespace extension { 
is it necessary?  conflict it not possible.
other purpose?

**Comment by owen200008** [comment]
it is compatibility. if no define it run the same with before

File: extensions/assets-manager/AssetsManagerEx.cpp (hunk 1 above)
Comment (minggo) [reply extensions/assets-manager/AssetsManagerEx.cpp:52]: yep, to avoid conflict. It is better to put it into an anonymous namespace.

**Pull request closed without merging**